packages/protocol/test/common/integration.ts

```diff
@@ -413,10 +425,19 @@ Array.from([
     const decimals = 18
 
     before(async () => {
-      exchange = await getDeployedProxiedContract(exchangeId, artifacts)
-      stableToken = await getDeployedProxiedContract(stableTokenId, artifacts)
-      multiSig = await getDeployedProxiedContract('ReserveSpenderMultiSig', artifacts)
-      reserve = await getDeployedProxiedContract('Reserve', artifacts)
+      exchange = await getDeployedProxiedContract(
+        exchangeId,
+        ArtifactsSingleton.getInstance('mento')
```

Comment:
Might be cleaner (not just in this file) to create a `mentoArtifacts` instance once and use throughout a file?

Or as mentioned previously above, have a unified wrapper.